packages/ics721-types/src/ibc.rs

```diff
@@ -0,0 +1,163 @@
+use cosmwasm_schema::cw_serde;
```

Comment:
Why is the other similar file called token_types but this one doesn't have _types in the name? Should be ibc_types no?

Reply:
First of all: before refactoing all types was in ics721 packages. I just moved it to dedicated `ics721-types` packages. Main reason was due to cyclic dependencies in https://github.com/arkprotocol/cw721-proxy/blob/main/Cargo.toml#L28

So `cw721-proxy` repo was using `ics721` repo and vice versa.

Will rename this file.

Reply:
renamed to `ibc_types.rs`